File: Data Structures & Algorithms/top-k-elements-in-list/submission-4.cpp
class Solution {
public:
    vector<int> topKFrequent(vector<int>& nums, int k) {
        map<int , int> mp;
    for (int i : nums)
    {
        if (mp.contains(i))
            mp[i]++;
        else
            mp[i]=1;
    }

    vector<pair<int,int>> tmp;
    for (auto it : mp)
        tmp.push_back(it);

    sort(tmp.begin(),tmp.end(),[](pair<int,int>p1,pair<int,int>p2)
    {
        return p1.second>p2.second;
    });
    vector<int>res;
    for (int i = 0; i < k; ++i)
    {
        res.push_back(tmp[i].first);
    }

    return res;
    }
};

File: Data Structures & Algorithms/is-anagram/submission-4.cs
public class Solution {
    public bool IsAnagram(string s, string t) {
      if(s.Length != t.Length) return false;

char[] s1=s.ToCharArray();
char[] t1=t.ToCharArray();

Array.Sort(s1);
Array.Sort(t1);

bool flag = false;

for (int i = 0; i < s1.Length; i++)
{ flag = s1[i]==t1[i];
 if (!flag) return false;
}


return flag;
    }
}
